javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/typesolvers/JarTypeSolver.java

```diff
@@ -74,14 +76,11 @@ private File dumpToTempFile(InputStream inputStream) throws IOException {
         byte[] buffer = new byte[8 * 1024];
 
         try {
-            OutputStream output = new FileOutputStream(tempFile);
-            try {
+            try (OutputStream output = new FileOutputStream(tempFile)) {
```

Comment:
can the try with variables have a finally? If that is the case we could combine the two try

Reply:
Yes, try-with-resources can have finally and catch blocks as usual, so yes, would definitely make sense to combine the two try blocks.